pages/blog/2024-april-summary.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Community Update: April 2024"
+date: 2024-04-30T06:00:00+01:00
+type: Communication
+tags:
+  - Project Status
+cover: /img/posts/2024-blog-banner/blog-banner-april.webp
+authors:
+  - name: Thulisile Sibanda
+    photo: /img/avatars/thulieblack.webp
+    link: https://www.linkedin.com/in/v-thulisile-sibanda/
+    byline: AsyncAPI Community Manager
+excerpt: 'Community update for April'
+featured: true
+---
+
+We are excited and commend [Specmatic](https://specmatic.in/) for their commitment to supporting open-source projects. They have sponsored [Pavel Bodiachevskii](https://www.linkedin.com/in/pavel-bo/), whose project assists in parsing AsyncAPI specs in Java and converting the parsed specs into executable contracts using the jasyncapi library. This kind of support is truly invaluable to the community and open-source maintainers. Please [check out the latest jasyncapi release](https://github.com/asyncapi/jasyncapi/releases/tag/1.0.0-RC) for more details.
+
+## AsyncAPI Conf on Tour
+
+We are thrilled to announce that AACoT'24 will return to London this year, and [the Call for Speakers is now open](https://conference.asyncapi.com/venue/London). Start preparing your proposals today, and get ready to join us in September!
+
+On an exciting note, we plan to host an online edition sometime this year. For more context, [please attend the upcoming community meeting](https://github.com/asyncapi/community/issues/1184).
```

Comment:
not sure we can say we plan,
it was just me and you having a convo and loved the idea, 

`we have an idea`?

Reply:
I have faith that why I used plan but yeah let use `idea` instead.😝

Reply:
sorry, my atheistic soul kicked in 😄